### ISSUE TITLE
Get back possibility to work with abstract page and panels without yaml

### ISSUE THREAD

**Issue by eliogrin** [Invictum/invictum-test#41]
**1st case:** 
HomePage extends AbstractPage (wich one without yaml file) we will get exception:

```
[main] ERROR com.github.invictum.utils.ResourceProvider - Unable to find 'AbstractPage.yaml' in '/target/test-classes/locators' directory

net.thucydides.core.pages.WrongPageError: The page object class com.*.pages.HomePage looks dodgy:
Failed to instantiate page (java.lang.IllegalStateException)
at com.github.invictum.EnhancedSerenityRunner.methodInvoker(EnhancedSerenityRunner.java:59)
```

**2nd case:**
HeaderPanel extends AbstractPanel (wich one without yaml file) we will get exception:

```
[main] ERROR com.github.invictum.utils.ResourceProvider - Unable to find 'AbstractPanel.yaml' in '/target/test-classes/locators' directory
java.lang.IllegalStateException
    at com.github.invictum.utils.ResourceProvider.getFile(ResourceProvider.java:36)
    at com.github.invictum.unified.data.provider.parsers.YamlParser.load(YamlParser.java:25)
    at com.github.invictum.unified.data.provider.UnifiedDataProviderFactory.mergeLocators(UnifiedDataProviderFactory.java:26)
    at com.github.invictum.unified.data.provider.UnifiedDataProviderFactory.getInstance(UnifiedDataProviderFactory.java:42)
    at com.github.invictum.panels.AbstractPanel.<init>(AbstractPanel.java:26)
    at com.*.panels.AbstractPanel.<init>(AbstractPanel.java:6)
```

**3rd case:** 
HeaderPanel extends AbstractPanel (wich one with yaml file but without locators in it)

```
java.lang.NullPointerException
    at com.github.invictum.unified.data.provider.UnifiedDataProviderFactory.mergeLocators(UnifiedDataProviderFactory.java:27)
    at com.github.invictum.unified.data.provider.UnifiedDataProviderFactory.getInstance(UnifiedDataProviderFactory.java:42)
    at com.github.invictum.panels.AbstractPanel.<init>(AbstractPanel.java:26)
    at com.*.panels.AbstractPanel.<init>(AbstractPanel.java:6)
    at com.*.panels.HeaderPanel.<init>(HeaderPanel.java:5)
```

**Comment by eliogrin** [comment]
BTW. Like a hack for this version of framework you can create empty yaml files for abstract panels or pages.

**Comment by Invictum** [comment]
Fixed. Will be available in 0.1.15 version.
